site/content/internal/infrastructure/vpn-cloud.md

```diff
@@ -37,12 +37,15 @@ Select `Show More` and hit `Download Profile`
    <span style="display:block;text-align:center;width:50%">![General settings Viscosity](/img/vpn_cloud_5.jpg)</span>
 
 - Go to Networking tab and update the DNS settings. 
-   * Select for the Mode to be `Split DNS`.  
+   * Select for the Mode to be `Full DNS`.  
    * As `Servers` put: `pritunl.core.cloud.mattermost.com` which is VPN's server IP.
    * In `Domains` put: `cloud.mattermost.com`, this will split traffic for those domains
     <span style="display:block;text-align:center;width:50%">![Network settings Viscosity](/img/vpn_cloud_6.jpg)</span>
+    
+6. Add, if it is not there, in your `/etc/resolv.conf`:
+    `nameserver 10.247.0.2`
```

Comment:
I haven't done this and don't have any issues, but I assume something must require it.

Reply:
It was failing on Eli's configuration, so I was thinking it's a good addition